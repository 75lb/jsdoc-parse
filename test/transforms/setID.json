[
    {
        "kind": "module",
        "longname": "module:human",
        "id": "module:human"
    },
    {
        "kind": "clive"
    },
    {
        "name": "module:human",
        "kind": "class",
        "meta": {
            "code": {
                "name": "Human"
            }
        },
        "longname": "module:human",
        "id": "module:human"
    },
    {
        "name": "Organ",
        "kind": "class",
        "longname": "module:human~Organ",
        "id": "module:human~Organ",
        "memberof": "module:human"
    },
    {
        "name": "Cell",
        "kind": "class",
        "longname": "module:human~Organ~Cell",
        "id": "module:human~Organ~Cell",
        "memberof": "module:human~Organ"
    },
    {
        "kind": "member",
        "longname": "module:human~Organ#redCell",
        "id": "module:human~Organ#redCell",
        "memberof": "module:human~Organ",
        "type": {
            "names": [
                "module:human~Organ~Cell"
            ]
        }
    },
    {
        "kind": "member",
        "longname": "module:human#liver",
        "id": "module:human#liver",
        "memberof": "module:human",
        "type": {
            "names": [
                "module:human~Organ"
            ]
        }
    },
    {
        "kind": "function",
        "longname": "module:human#getOrgan",
        "id": "module:human#getOrgan",
        "memberof": "module:human",
        "returns": [
          {
            "type": {
              "names": [
                "module:human~Organ"
              ]
            }
          }
        ]
    }
]
