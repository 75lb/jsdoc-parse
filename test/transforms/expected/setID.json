{
	"id": "the original longname",
	"longname": "the original longname"
}
